openapi.yaml

```diff
@@ -10615,6 +10618,300 @@ components:
+        content:
+          $ref: '#/components/schemas/WAContent'
+      required:
+        - type
+        - name
+        - content
+    WAContent:
+      title: WAContent
+      x-stoplight:
+        id: fklmr2skqadi5
+      type: object
+      description: 'The content of your message which may contain any text and media that will be sent to the recipient. The contents can include a header, a body, and buttons.'
+      properties:
+        header:
+          $ref: '#/components/schemas/WAContentHeader'
+        body:
+          type: array
+          description: The contents of your WhatsApp message
+          items:
+            title: Object
+            type: object
+        button:
+          type: array
+          description: |-
+            The contents of your dynamic button.
+
+            If the button subtype is “url” you will need the following parameters:
+            “Index”, “sub_type”, “text”
+
+            If the button subtype is a “quick reply”, you will need the following parameters:
+            “Index”, “sub_type”, “payload"
+          items:
+            $ref: '#/components/schemas/WAContentButton'
+      required:
+        - body
+      x-examples:
+        Example 1:
+          header:
+            image:
+              link: 'http://placekitten.com/200/300'
+          body:
+            - text: John
+            - text: ACTIVE
+            - text: '+619702873117'
+          button:
+            - index: '0'
+              sub_type: quick_reply
+              payload: Yes-Button-Payload
+            - index: '1'
+              sub_type: quick_reply
+              payload: No-Button-Payload
+    WAContentHeader:
+      title: WAContentHeader
+      x-stoplight:
+        id: hevlc9qwqt4y3
+      type: object
+      x-examples:
+        Image Example:
+          header:
+            image:
+              link: 'http://placekitten.com/200/300'
+        Video Example:
+          video:
+            link: 'https://www.learningcontainer.com/wp-content/uploads/2020/05/sample-mp4-file.mp4'
+        Document Example:
+          document:
+            link: 'https://s3-eu-west-1.amazonaws.com/hughhopkins.co.uk/sample_media/Sf+flight+itinerary.pdf'
+        Location Example:
+          location:
+            longitude: -122.425332
+            latitude: 37.758056
+            name: Facebook HQ
+            address: '1 Hacker Way, Menlo Park, CA 94025'
+      description: 'The header of your message can include one type of media content e.g. image, video, document and location.'
+      properties:
+        image:
+          title: Object
+          type: object
+          description: Link to image location
+        video:
+          title: Object
+          type: object
+          description: Link to video location
+        document:
+          title: Object
+          type: object
+          description: Link to document location
+        location:
+          title: Object
+          type: object
```

Comment:
What are the key-value pairs required for these objects:

* `image`
* `video`
* `document`
* `location`

Could we please add the appropriate `properties` for each of these.

Reply:
added properties to the WAContentHeader objects